scripts/get_archive

```diff
@@ -37,7 +36,7 @@ while [ ${NBWGET} -gt 0 ]; do
 
       [ -z "${PKG_SHA256}" -o "${PKG_SHA256}" = "${CALC_SHA256}" ] && break 2
 
-      printf "%${BUILD_INDENT}c $(print_color CLR_WARNING "WARNING") Incorrect checksum calculated on downloaded file: got ${CALC_SHA256} wanted ${PKG_SHA256}\n\n" ' '>&$SILENT_OUT
+      build_msg "CLR_WARNING" "WARNING" "Incorrect checksum calculated on download file: got ${CALC_SHA256} wanted ${PKG_SHA256}"
```

Comment:
This is losing one `\n` - intentional? And same below.

Reply:
Intentional for both. I don't see value in the extra blank line. Easily readded as part of $3 if desired.

Reply:
Ok fine, let's see how it goes without the extra `\n` (sometimes I prefer a little bit of extra space for clarity).